modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/HierarchyDataRenderer.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+export function HierarchyDataRenderer({value}: IProps) {
+	const statusInfo = {
+		className: value ? 'label-info' : 'label-success',
+		label: value
+			? Liferay.Language.get('child')
+			: Liferay.Language.get('parent'),
+	};
+
+	return (
+		<strong className={`label ${statusInfo?.className}`}>
+			{statusInfo?.label}
+		</strong>
```

Comment:
Hey @p-albuquerque, I think we can use [Clay Label](https://clayui.com/docs/components/label.html) component here instead of a strong one with a custom styled className, the reason for this is because if updates come from the infra-frontend team in case they decide to update this component in the future, we will also have our updated components without work